libraries/chain/asset_evaluator.cpp

```diff
@@ -404,7 +404,7 @@ void_result asset_update_bitasset_evaluator::do_apply(const asset_update_bitasse
       // feeds must be reset if the backing asset is changed
       bool backing_asset_changed = false;
       bool is_witness_or_committee_fed = false;
-      if (op.new_options.short_backing_asset != bitasset_to_update->options.short_backing_asset)
+      if (db().head_block_time() >= HARDFORK_CORE_868_TIME && op.new_options.short_backing_asset != bitasset_to_update->options.short_backing_asset)
```

Comment:
This line seems a bit too long. Perhaps add a line wrap.

Reply:
Done